app/models/setting.model.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, Sequelize) => {
+    const Setting = sequelize.define("setting", {
+      storyId: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
```

Comment:
I don't think we need to reference the storyId in this model. Probably just make it an auto incrementing i"?
```suggestion
  id: {
    type: DataTypes.INTEGER,
    primaryKey: true,
    autoIncrement: true
  },
```